app/forms/LocationFormProvider.scala

```diff
@@ -39,8 +38,8 @@ sealed trait LocationFormProvider extends Mappings {
     )
 }
 
-class LoadingLocationFormProvider @Inject() (implicit phaseConfig: PhaseConfig) extends LocationFormProvider {
-  override val locationMaxLength: Int = phaseConfig.values.loadingLocationMaxLength
+class LoadingLocationFormProvider @Inject() extends LocationFormProvider {
```

Comment:
Can we just have a single class say LocationFormProvider in here seeing as the loadingLocationMaxLength and unloadingLocationMaxLength resolve the same value of 35?